apps/ui/src/networks/offchain/actions.ts

```diff
@@ -160,26 +161,17 @@ export function createActions(
 
       const name = strategiesNames[0];
       const strategy = getOffchainStrategy(name);
-      if (!strategy) return [{ address: name, value: 0n, decimals: 0, token: null, symbol: '' }];
-
-      const result = await strategy.getVotingPower(
-        spaceId,
-        voterAddress,
-        strategiesOrValidationParams,
-        snapshotInfo
-      );
-
-      if (strategy.type !== 'remote-vp') {
-        return [
-          {
-            address: strategiesNames[0],
-            decimals: 0,
-            symbol: '',
-            token: '',
-            chainId: snapshotInfo.chainId,
-            value: result[0]
-          }
-        ];
```

Comment:
We lost this case, now it throws when trying to propose:
http://localhost:8080/#/s:stgdao.eth/create

Reply:
New change doesn't really cover this, it will now show address as `undefined` instead of `strategiesNames[0]`.

Reply:
@wa0x6e